Tests/Engine/CustomizedRule.tests.ps1

```diff
@@ -66,7 +66,7 @@ Describe "Test importing correct customized rules" {
 				$null,"Wow6432Node" | ForEach-Object {
 					try
 					{
-						Set-ItemProperty -Name "DisablePromptToUpdateHelp" -Path "HKLM:\SOFTWARE\$($_)\Microsoft\PowerShell" -Value 1 -Force
+						Set-ItemProperty -Name "DisablePromptToUpdateHelp" -Path "HKLM:\SOFTWARE\$($_)\Microsoft\PowerShell" -Value 1 -Force -EA SilentlyContinue
```

Comment:
Minor nit: Use `-ErrorAction SilentlyContinue` for consistency, readability and to not rely on parameter aliases